lib/commands/context.js

```diff
@@ -70,6 +70,7 @@ extensions.getNewRemoteDebugger = async function getNewRemoteDebugger () {
   }
   return createRemoteDebugger({
     bundleId: this.opts.bundleId,
+    additionalBundleIds: this.opts.additionalWebviewBundleIds,
```

Comment:
this.opts.additionalWebviewBundleIds || [] ?

Reply:
This is done on the remote debugger side.